plugins/modules/identity/bitwarden.py

```diff
@@ -0,0 +1,699 @@
+    - Folders in Bitwarden have no duplicate names per organization.
+    - The name of items in folders uniquely identifies the items per organization.
+notes:
+    - Tested with C(bw) version 2022.6.2
+short_description: Manipulate items in Bitwarden
+description:
+    - M(community.general.bitwarden) wraps the C(bw) command line utility to modify Bitwarden-compatible vaults.
+    - A fatal error occurs if any of the items being searched for can not be found.
+    - Recommend using with the C(no_log) option to avoid logging the values of the secrets being retrieved.
+options:
+    cli_path:
+        type: path
+        description: Used to specify the exact path to the C(bw) command line interface
+        required: false
+        default: 'bw'
+    organization_name:
+        type: str
+        description:
+            - Name of organization to target.
+            - Leave as null for a wildcard vault or C('') for personal vaults.
+            - Mutually exclusive from organization_id.
+        required: false
+        default: null
+    organization_id:
+        type: str
+        description:
+            - UID of organization to target.
+            - Leave as null for a wildcard vault.
+            - Mutually exclusive from organization_name.
+        required: false
+        default: null
+    folder_name:
+        type: str
+        description:
+            - Name of folder to target.
+            - Leave as null for a wildcard folder.
+            - Use C('') or C('No Folder') for no folder.
+            - Mutually exclusive from folder_id.
+        required: false
+        default: null
+    folder_id:
+        type: str
+        description:
+            - UID of folder to target.
+            - Leave as null for a wildcard folder.
+            - Mutually exclusive from folder_name.
+        required: false
+        default: null
+    item_name:
+        type: str
+        description:
+            - Name of item to target.
+            - Mutually exclusive from item_id.
+        required: false
+        default: null
+    item_id:
+        type: str
+        description:
+            - UID of item to target.
+            - Mutually exclusive from item_name.
+        required: false
+        default: null
+    login:
+        type: dict
+        description:
+            - Mapping of keys to values for bw login details such as C(username) and C(password).
+        required: false
+        default: {}
+    notes:
+        type: str
+        description:
+            - A note about the item that is stored as metadata about the item.
+        required: false
+        default: null
+    target:
+        type: str
+        description:
+            - The type of item being added/modified/removed/queried.
+            - For additions/modifications/removal, must be one of C(item) or C(folder).
+            - For queries, the above targets are permitted along with
+                - C(items),
```

Comment:
This formatting will not work. Rewrite this in a way that it is one long string of text that can be broken into lines at arbitrary points, like
```
    - For queries, the above targets are permitted along with C(items), C(folders), ...
```